nix/haskell.nix

```
@@ -37,28 +45,12 @@ let
           # "stm" "terminfo"
         ];
       }
+      { packages = (lib.listToAttrs (map (p: lib.nameValuePair
+        # Compile all local packages with -Werror:
+        p { configureFlags = [ "--ghc-option=-Werror" ]; }
+      ) projectPackages));}
```

I find this way easier to read:

```suggestion
      # Compile all local packages with -Werror:
      { packages = lib.genAttrs (name: { configureFlags = [ "--ghc-option=-Werror" ]; }); }
```